src/transforms/hexbin.d.ts

```diff
@@ -4,8 +4,8 @@ import type {Initialized} from "./basic.js";
 /** Options for the hexbin transform. */
 export interface HexbinOptions {
   /**
-   * The distance between centers of neighboring (pointy-topped) hexagons, in
-   * pixels; defaults to 20.
+   * The horizontal distance between centers of neighboring (pointy-topped)
+   * hexagons, in pixels; defaults to 20.
```

Comment:
_The distance between centers of neighboring hexagons_ was complete and correct :)

Reply:
Of course! Oops. Thanks for the correction.